.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    target-branch: "master"
+    schedule:
+      interval: "daily"
+
+    ignore:
+      - dependency-name: "amazon.ion"
+      - dependency-name: "ionhash"
```

Comment:
ionhash isn't likely to be shared so it shouldn't be a part of the ignorelist

Reply:
We don't know for sure. To err on the side of caution, we should ignore